### PR TITLE
Add submission flow

### PR DESCRIPTION
**Pull request by freekmurze**
This PR adds a submission flow:

- a guest can submit a YouTube ID and email (we only accept valid YouTube IDs of upcoming streams that aren't in the db yet)
- a mail will be sent to Christoph with to signed links: one to approve the stream, one to reject the stream
- when a stream is approved, a confirmation mail will be sent to the submitted email

To do:
- currently there is no UI to submit the YouTube ID / email. An endpoint to receive the submission is already present (SubmitStreamController)
- a few tests have been added that prove that the logic is ok, but I didn't do a manual test myself to make 100% sure that this all works.

### CODE REVIEW

File: resources/views/pages/streamRejected.blade.php [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<x-main-layout>
+    <h1>Stream approved!</h1>
```
Comment (noplanman) [inline resources/views/pages/streamRejected.blade.php:2]: `Stream rejected!`

File: resources/views/mail/approved.blade.php [hunk 2]
```diff
@@ -0,0 +1,10 @@
+@component('mail::message')
+Hi,
+
+Your stream [{{ $stream->title }}]({{ $stream->url() }}) was approved. You can now view it on [on our homepage]({{ route('home') }})
```
Comment (noplanman) [inline resources/views/mail/approved.blade.php:4]: view it on [~on~ our homepage]

**Review by christophrumpel — APPROVED** [review]
Thanks for your effort. I added the missing parts and refactored a bit to use a LiveWire component.

Do you like to give it a look? If not, I'll just merge it to make it live :-)

**Comment by christophrumpel** [comment]
Thanks a lot, @freekmurze and @noplanman  👏

**Pull request merged**